resources/views/partials/forms/edit/location-select.blade.php

```diff
@@ -4,13 +4,12 @@
     {{ Form::label($fieldname, $translated_name, array('class' => 'col-md-3 control-label')) }}
     <div class="col-md-7{{  ((isset($required) && ($required =='true'))) ?  ' required' : '' }}">
         <select class="js-data-ajax" data-endpoint="locations" data-placeholder="{{ trans('general.select_location') }}" name="{{ $fieldname }}" style="width: 100%" id="{{ $fieldname }}_location_select" aria-label="{{ $fieldname }}" {!!  ((isset($item)) && (Helper::checkIfRequired($item, $fieldname))) ? ' data-validation="required" required' : '' !!}{{ (isset($multiple) && ($multiple=='true')) ? " multiple='multiple'" : '' }}>
-            @if ($location_id = old($fieldname, (isset($item)) ? $item->{$fieldname} : ''))
-                <option value="{{ $location_id }}" selected="selected" role="option" aria-selected="true"  role="option">
-                    {{ (\App\Models\Location::find($location_id)) ? \App\Models\Location::find($location_id)->name : '' }}
-                </option>
-            @else
-                <option value=""  role="option">{{ trans('general.select_location') }}</option>
-            @endif
+            {{  ((isset($multiple)) && ($multiple=='true')) ? " multiple='multiple'" : '' }}
+                @if($location_id = old($fieldname, (isset($item)) ? $item->{$fieldname} : ''))
```

Comment:
This `@if` isn't closed properly. I'm not sure if it's working because `@endisset` is closing it but it should have a matching `@endif` added.

(the other selects in this PR should be updated as well)